package.json

```diff
@@ -12,14 +12,17 @@
     "prettier": "prettier --write ."
   },
   "dependencies": {
+    "@cypress/webpack-preprocessor": "^6.0.0",
     "@ministryofjustice/frontend": "1.8.0",
-    "govuk-frontend": "^4.4.0"
+    "govuk-frontend": "^4.4.0",
+    "path-browserify": "^1.0.1",
+    "webpack": "^5.89.0"
```

Comment:
I think these new packages should be in devDependencies since we don't roll them into our release code.

Reply:
Good point. However, I notice that yarn.lock doesn't discriminate dev from runtime dependencies. Do we ever rely on yarn to install our dependencies at any point from yarn.lock with no reference to package.json; if so, will it install our dev dependencies too?

Reply:
Just tried this locally and it only installed the prod dependencies. I guess it uses `package.json` to determine which packages to install, and the lockfile only for versions.